test/test_samples.py

```diff
@@ -1,11 +1,21 @@
+import platform
+
 import pytest
 
 
+@pytest.mark.skipif(
+    platform.system() == "Windows",
+    reason=r"python: can't open file 'D:\a\samples\sample_capture_from_file.py': [Errno 2] No such file or directory",
```

Comment:
Maybe you can make an issue to investigate this further later.

Reply:
https://github.com/zivid/zivid-python/issues/82

Reply:
It is hard to find the link between this code and #82.
 * #82 should mention that these two tests should be re-enabled.
 * The `reason` here should reference the #82.